<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticsForm</class>
 <widget class="QWidget" name="StatisticsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>900</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="themeLabel">
       <property name="text">
        <string>Theme:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="themeComboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="animatedLabel">
       <property name="text">
        <string>Animation:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="animatedComboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="legendLabel">
       <property name="text">
        <string>Legend:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="legendComboBox"/>
     </item>
     <item>
      <widget class="QCheckBox" name="antialiasCheckBox">
       <property name="text">
        <string>Anti-aliasing</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>themeComboBox</tabstop>
  <tabstop>animatedComboBox</tabstop>
  <tabstop>legendComboBox</tabstop>
  <tabstop>antialiasCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>themeComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>StatisticsForm</receiver>
   <slot>updateUI()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>legendComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>StatisticsForm</receiver>
   <slot>updateUI()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>379</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>animatedComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>StatisticsForm</receiver>
   <slot>updateUI()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>antialiasCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>StatisticsForm</receiver>
   <slot>updateUI()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateUI()</slot>
 </slots>
</ui>
